src/content/components/BugList/BugList.js

```diff
@@ -80,41 +74,64 @@ const priorityOptions = [
   },
 ];
 
-export class BugList extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      selectedBugs: {},
-      showResolved: props.showResolved,
-      editSelection: -1,
-      newIteration: currentIteration,
-      newPriority: "---",
+export const BugList = ({ ...props }) => {
```

Comment:
`{ ...props }` doesn't make sense. just use `props` or else spread out all the main props like `{ showResolved, bugs, ...props }`